src/platforms/android/common/proguard.mdx

```diff
@@ -40,7 +40,7 @@ buildscript {
 }
 ```
 
-The plugin will automatically generate appropriate ProGuard mapping files and upload them when you run `gradle assemble{BuildVariant}`. For example, `assembleRelease` — Release is the default, but the plugin works for others if you have enabled ProGuard/R8. The credentials for the upload step are loaded via environment variables or from a `sentry.properties` file in your project root.
+The plugin will automatically generate appropriate ProGuard mapping files and upload them when you run `gradle assemble{BuildVariant}`. For example, `assembleRelease` — Release is the default, but the plugin works for others if you have enabled ProGuard/R8. The credentials for the upload step are loaded via environment variables or from a `sentry.properties` file in your Android project root.
```

Comment:
this is wrong, the project root isn't necessarily an Android project.
A Gradle project with many projects Java/Android/Groovy etc isn't an Android project root and still requires to be in the project root folder.
eg https://github.com/getsentry/sentry-java/blob/main/sentry.properties

Reply:
Isn't it the Android doc?

Reply:
yes, but you may have an android module inside of a bigger Gradle solution that may have different projects, the current description fits both use cases.